mesonbuild/scripts/run_tool.py

```diff
@@ -3,17 +3,86 @@
 
 from __future__ import annotations
 
-import itertools
+import asyncio
+import asyncio.subprocess
```

Comment:
You don't need to import both of these -- importing a submodule implies importing the top-level module too. :)

Reply:
I find it clearer to import both but it's the same. :)